e2e/Makefile

```diff
@@ -3,7 +3,7 @@ GOFLAGS=-mod=vendor
 export GOFLAGS
 GINKGO=$(GOPATH)/bin/ginkgo
 KUBECTL=/usr/local/bin/kubectl
-KUBERNETES_VERSION=1.14.3
+KUBERNETES_VERSION=1.15.0
```

Comment:
Should we use 1.15.3?
kind can also be updated to v0.5.1.

Reply:
This is used in line 41 and when we checked, `kindest/node:v1.15.3` didn't exist.
However, as the image was made yesterday, we use `KUBERNETES_VERSION=1.15.3`